podpac/core/settings.py

```diff
@@ -19,8 +17,15 @@
 # Settings Defaults
 DEFAULT_SETTINGS = {
     'DEBUG': False,
-    'CACHE_DIR': None,
-    'CACHE_TO_S3': False,
+    'DEFAULT_CACHE': ['ram'],
+    'CACHE_OUTPUT_DEFAULT': True,
+    'RAM_CACHE_MAX_BYTES': 1e9, # ~1GB
+    'DISK_CACHE_MAX_BYTES': 10e9, # ~10GB
```

Comment:
@jmilloy @drc-creare  Are these limits implemented?

Reply:
No, I guess we should remove them or comment them out for now.